setup.py

```diff
@@ -18,10 +18,10 @@ def get_version():
     raise SystemExit("Could not find version string.")
 
 
-if sys.version_info < (3, 8):
-    sys.exit('Instaloader requires Python >= 3.8.')
+if sys.version_info < (3, 9):
+    sys.exit('Instaloader requires Python >= 3.9.')
 
-requirements = ['requests>=2.4']
+requirements = ['requests>=2.32']
```

Comment:
```suggestion
requirements = ['requests>=2.25']
```

I suggest to require >= 2.25 since this is safely recent-enough for us, but old enough to be shipped in older, but still-supported debian or ubuntu LTS releases.